cli_tools/gce_vm_image_import/importer/importer.go

```diff
@@ -123,7 +123,11 @@ func validateSourceFile(storageClient domain.StorageClientInterface, sourceBucke
 	byteCountingReader := daisycommon.NewByteCountingReader(rc)
 	// Detect whether it's a compressed file by extracting compressed file header
 	if _, err = gzip.NewReader(byteCountingReader); err == nil {
-		return daisy.Errf("the input file is a gzip file, which is not supported by 'gcloud compute images import'. To import a file that was exported from Google Compute Engine, please use 'gcloud compute images create' instead")
+		return daisy.Errf("the input file is a gzip file, which is not supported by " +
```

Comment:
Since image import wrapper will now be used via API as well, referring to gcloud commands in errors will be confusing for API users. Can we refer to specific operations in a generic way (image import, image create) instead of referring to commands? We can also provide links where gcloud/API/UI steps are documented.

Reply:
sgtm!